spec/index.html

```diff
@@ -232,65 +232,122 @@
   }
 </style>
 </head>
-  <body>
-    <section id='abstract'>
-      <p>
-        In recent years, YAML emerged as a more concise format
-        to represent information serialized as JSON,
-        including Linked Data.
-        This document defines how to serialize linked data
-        in YAML.
-        Moreover, it registers the application/ld+yaml media type.
-      </p>
-    </section>
-
-    <section id='sotd'>
-      <p>
-        This is an unofficial proposal.
-      </p>
-    </section>
+<body>
+  <section id='abstract'>
+    <p>
+      In recent years, [[YAML]] emerged as a more concise format
+      to represent information that had previously been serialized as JSON,
+      including Linked Data.
+      This document defines how to serialize linked data
+      in YAML.
+      Moreover, it registers the `application/ld+yaml` media type.
+    </p>
+  </section>
 
-    <section>
-      <h2>Introduction</h2>
-      <p>
-        JSON-LD is a JSON-based format to serialize Linked Data.
-        In recent years, YAML emerged as a more concise format
-        to represent information serialized as JSON,
-        including API specification, data schemas, and Linked Data.
-      </p>
-      <p>
-        Since YAML is more expressive than JSON,
-        both in the available data types and in the document structure
-        (see [[I-D.ietf-httpapi-yaml-mediatypes]]),
-        this document identifies constraints on YAML
-        such that it can be used to represent JSON-LD documents.
-      </p>
-    </section>
+  <section id='sotd'>
+    <p>This document has been developed by the
+      <a href="https://www.w3.org/community/json-ld/">JSON-LD Community Group</a>.</p>
+  </section>
 
-  <section id="conformance">
-    <p>A <a>YAML-LD document</a> complies with this specification if ...</p>
-    <p class="ednote">Define <dfn>YAML-LD document</dfn> somewhere.</p>
+  <section>
+    <h2>Introduction</h2>
     <p>
-      The term media type is imported from [[RFC6838]].
+      [[JSON-LD]] is a JSON-based format to serialize Linked Data.
+      In recent years, [[YAML]] emerged as a more concise format
+      to represent information that had previously been serialized as [[JSON]],
+      including API specification, data schemas, and Linked Data.
```

Comment:
```suggestion
      including API specifications, data schemas, and Linked Data.
```

Reply:
"API specification" is singular; "data schemas" and "Linked Data" are plural.  I don't think it makes sense to change the plurals to singulars, hence my suggested change.

Reply:
@gkellogg -- "API specification" is singular; "data schemas" and "Linked Data" are plural.  I don't think it makes sense to change the plurals to singulars, hence my suggested change.